addons/website/static/src/js/tours/homepage.js

```diff
@@ -64,4 +65,5 @@ registerThemeHomepageTour("homepage", () => [
     ...insertSnippet(snippets[5]),
     ...insertSnippet(snippets[6]),
     ...insertSnippet(snippets[7]),
+    ...goToTheme(),
```

Comment:
In my theme repo review I said:
>  It could be automatic in the registerThemeHomepageTour function definition in website?

Here you don't do that: you only change the definition of the tour for the community repo. You still have to change it automatically for all themes tours (did you run the theme test to confirm it actually checks the bug is solved? I don't think it does... if you run the `test_02_homepage_tour_every_theme` without your CSS change in the themes, it probably passes because it still does not test going to the theme tab on all themes)

So here I am thinking about actually changing the `registerThemeHomepageTour` function itself, to actually add the `goToTheme` steps at the end automatically.

Reply:
(and it has to be register under the "homepage" name as this is the one that should be displayed to users after the configurator is followed)